### ISSUE TITLE
[Cata] Paragon information related errors in 4.4.1 client

### ISSUE THREAD

**Issue by Road-block** [tvlfrosty/Reputation_Guide#88]
The 4.4.1 client now has the C_Reputation namespace but there is no 100% parity with the methods it contains between retail and cataclysm classic client.

Additionally the `ReputationDetailAtWarCheckBox` reference errors.
The latter is an easy fix by adding 
```lua
----------------------------------------------
-- 07 UI Changes based on current expansion --
----------------------------------------------
function REP:MakeUIChanges()
  if REP.IsClassic or REP.IsCata then
  --<snipped>
```
at line 850 or thereabouts.

The other errors I fixed in my local copy with
```lua
  REP_Orig_GetFactionParagonInfo = C_Reputation and C_Reputation.GetFactionParagonInfo or function() return end
  REP_Orig_IsFactionParagon = C_Reputation and C_Reputation.IsFactionParagon or function() return end
  REP_Orig_GetFriendshipReputation = C_Reputation and C_GossipInfo.GetFriendshipReputation or function() return end
```

**Comment by tvlfrosty** [comment]
I will have a look right away, haven't had the time to log into cata since new patch.

**Comment by tvlfrosty** [comment]
Should be fixed in the "1.3.1-release" on curse, thanks for letting me know it broke with the 4.4.1 patch!
